File: Java/OPSJLib/src/ops/Listener.java
/*
 * To change this template, choose Tools | Templates
 * and open the template in the editor.
 */

package ops;

/**
 *
 * @author Anton Gravestam
 */
public interface Listener<T>
{
    public void onNewEvent(Notifier<T> notifier, T arg);

}

File: C#/Ops/Listener.cs
///////////////////////////////////////////////////////////
//  Listener.cs
//  Implementation of the Interface Listener
//  Created on:      12-nov-2011 09:25:31
//  Author:
///////////////////////////////////////////////////////////

namespace Ops 
{
	public interface IListener<T>
    {
		void OnNewEvent(Notifier<T> notifier, T arg);
	}

}